.github/actions/ci-setup/action.yaml

```diff
@@ -2,7 +2,7 @@ name: "CI setup"
 inputs:
   node-version:
     description: "Node version"
-    default: 18.14.1
+    default: 18.18.0
```

Comment:
In order to use node's fetch I had to update to a newer version which had a bugfix that I needed.